openpype/hosts/houdini/hooks/set_default_display_and_view.py

```diff
@@ -0,0 +1,68 @@
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class SetDefaultDisplayView(PreLaunchHook):
+    """Set default view and default display for houdini via OpenColorIO.
+
+    Houdini's defaultDisplay and defaultView are set by
+    setting 'OCIO_ACTIVE_DISPLAYS' and 'OCIO_ACTIVE_VIEWS'
+    environment variables respectively.
+
+    More info: https://www.sidefx.com/docs/houdini/io/ocio.html#set-up
+    """
+
+    app_groups = {"houdini"}
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+
+        OCIO = self.launch_context.env.get("OCIO")
+
+        # This is a cheap way to skip this hook if either global color
+        # management or houdini color management was disabled because the
+        # OCIO var would be set by the global OCIOEnvHook
+        if not OCIO:
+            return
+
+        houdini_color_settings = \
+            self.data["project_settings"]["houdini"]["imageio"]["workfile"]
+
+        if not houdini_color_settings["enabled"]:
+            self.log.info(
+                "Houdini workfile color management is disabled."
+            )
+            return
+
+        # This is a way to get values specified by admins if they already
+        # added 'OCIO_ACTIVE_DISPLAYS', 'OCIO_ACTIVE_VIEWS' manually
+        # using Ayon global env vars or Ayon app env vars
+        # or Ayon houdini tool
+        OCIO_ACTIVE_DISPLAYS = self.launch_context.env.get(
```

Comment:
if you don't use this unless `default_display` is set on the settings, this variable should go inside the `if default_display`